chat-plugins/wifi.js

```diff
@@ -82,26 +81,26 @@ var QuestionGiveAway = (function () {
 		return;
 	};
 
-	QuestionGiveAway.prototype.change = function (key, value, user, output) {
+	QuestionGiveAway.prototype.change = function	(key, value, user, output) {
 		if (user.userid !== this.host.userid) return this.sendReply("You cannot edit this giveaway. Only the host/starter can edit.");
 		if (this.phase !== 'pending') return output.sendReply("The giveaway has started, and the question/answer cannot be changed.");
 
 		switch (key) {
-		case 'question':
-			this.question = value;
-			output.sendReply("The question has been changed to " + value + ".");
-			break;
-		case 'answer':
-			var ans = this.sanitizeAnswers(value);
-			if (Object.keys(ans).length <= 0) return output.sendReply("You must specify at least one answer and must not contain any special characters.");
-
-			this.answers = ans;
-			output.sendReply("The answers have been changed to " + value + ".");
-			break;
+			case 'question':
+				this.question = value;
+				output.sendReply("The question has been changed to " + value + ".");
+				break;
+			case 'answer':
+				var ans = this.sanitizeAnswers(value);
+				if (Object.keys(ans).length <= 0) return output.sendReply("You must specify at least one answer and must not contain any special characters.");
+
+				this.answers = ans;
+				output.sendReply("The answers have been changed to " + value + ".");
+				break;
```

Comment:
Remove those added indents you put in there. `case` keywords should be in line with the `switch` statement.